### PR TITLE
Stable release of Front door API

### PR DESCRIPTION
**Pull request by johsun-msft**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by johsun-msft** [comment]
Original review already signed off at private branch https://github.com/Azure/azure-rest-api-specs-pr/pull/708

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5584 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5584/commits/3fa2db9141dddab09eadf62de4b260533193866b

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/FrontDoorWebApplicationFirewallPolicies": {
+      "get": {
+        "tags": [
+          "ListWebApplicationFirewalPolicies"
```
Comment (nschonni) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]:
```suggestion
          "ListWebApplicationFirewallPolicies"
```

**Comment by sergey-shandar** [comment]
https://github.com/Azure/azure-rest-api-specs-pr/pull/708

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+reverse-proxy route to accelerate traffic, detect backend availability, 
+and balance between healthy members of the pool.
+
+## Generated object-model UML diagram
+This diagram is generated from the swagger spec by "[`oav`](https://github.com/Azure/oav) `generate-uml`"
+![Generated UML snapshot](./assets/frontdoor.svg)
+
+## Swagger and JSON specification reference and tools
+ * [OpenAPI specification v2.0 (aka Swagger 2.0)](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md)
+ * [Swagger.io online editor](https://editor.swagger.io/)
```
Comment (sergey-shandar) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/README.md]: The tag must be added. The tag must include all OpenAPI files for this API version.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/README.md (hunk 2 above)
Comment (johsun-msft) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/README.md]: tag is already specified under specification\frontdoor\resource-manager\readme.md

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**